modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -16,139 +17,142 @@ The app monitoring dashboard helps you answer questions about your integrated ap
+=== View Application Metrics
 
-If your application runs on a cluster or server group, the aggregated metrics of the entire set of servers included in the server group or cluster are charted as a single plot line.
+To view all graphs for different time ranges, select the time interval in the top-right corner:
 
-[TIP]
-If you run your app on a cluster or server group and want to see the metrics at the individual server level, you can look at the dashboard for the individual server. If multiple apps are running on the server, you see the overall server performance, without a distinction between each of the apps running on it.
+* Built-in Anypoint Monitoring dashboard:
++
+image::monitoring-dashboards-time-ranges.png[]
+* Classic application monitoring dashboard:
++
+image::monitoring-dashboards-cbd92.png[]
 
-== Alerts
+If your application runs on multiple workers, the workers are charted as different curves on the same graphs, differentiated by different colors.
 
-Set up xref:alerts-on-runtime-manager.adoc#conditions-on-locally-deployed-applications[alerts] for applications and servers to notify you automatically whenever any of these metrics reaches a critical value for a sustained period of time.
+If your application runs on a server group or cluster, the aggregated metrics of the servers in the group or cluster appear as a single plot line on each graph. If you want to see the metrics at the individual server level, you can look at the dashboard for the individual server. If multiple apps are running on the server, you see the overall server performance, without a distinction between each of the apps running on it.
 
-== Access the Monitoring Dashboard for Servers
+== Access the Server Monitoring Dashboard
 
-[NOTE]
-The Anypoint Platform Private Cloud Edition doesn't support the monitoring dashboard feature.
+The server monitoring dashboard displays high-level information about the servers, including heap memory and CPU percentage, as well as detailed metrics for memory usage.
 
-. In Runtime Manager, in the menu on the left, click *Servers*. 
-. Click the row for the server to open the management panel on the right side of the page.
-. Click the *View Dashboard* button. +
-The dashboard for the server is displayed.
+To access the server monitoring dashboard:
 
+. In Runtime Manager, click *Servers*. 
+. Click the name of the server to monitor. 
++
+The dashboard for the server appears:
++
 image::monitoring-dashboards-server.png[]
 
-You can also navigate to a server's dashboard if you first open the dashboard for a cluster or server group that contains the server, then click the name of the server.
+You can also navigate to a server's monitoring dashboard from the <<Access the Server Group or Cluster Dashboard,server group or cluster dashboard>> that includes the server.
 
-== View Server Metrics
+=== View Server Metrics
```

Comment:
Would it be more precise to title this Methods of Viewing Server Metrics, and then start with some text about how you can display different views (overlay, specific period, and so on) depending using different parts of the UI?